src/adapter/src/coord/message_handler.rs

```diff
@@ -351,6 +351,15 @@ impl Coordinator {
                     active_subscribe.process_response(response);
                 }
             }
+            ControllerResponse::CopyToResponse(sink_id, response) => {
+                if let Some(active_copy_to) = self.active_copy_tos.get_mut(&sink_id) {
```

Comment:
The reason it's not, because we still need the state here, because `drop_compute_sinks` will be called later via `clear_connection` and that method drops the compute collections (it needs this entry to be available there) and then clears the state.

Reply:
(Resolving since we addressed this in slack)